src/svm/transactionUtils.ts

```diff
@@ -76,5 +116,33 @@ export async function sendTransactionWithLookupTable(
   versionedTx.sign([sender]);
   const txSignature = await connection.sendTransaction(versionedTx);
 
-  return { txSignature, lookupTableAddress };
+  // Confirm the versioned transaction
+  await confirmTransaction(connection, txSignature);
```

Comment:
This might conflict once #811 is merged in, but seems better to have reusable transaction confirmation helper